server/src/test/java/org/opensearch/common/cache/stats/DefaultCacheStatsHolderTests.java

```diff
@@ -127,49 +127,58 @@ public void testCount() throws Exception {
     }
 
     public void testConcurrentRemoval() throws Exception {
-        List<String> dimensionNames = List.of("dim1", "dim2");
+        List<String> dimensionNames = List.of("A", "B");
         DefaultCacheStatsHolder cacheStatsHolder = new DefaultCacheStatsHolder(dimensionNames, storeName);
 
         // Create stats for the following dimension sets
-        List<List<String>> populatedStats = List.of(List.of("A1", "B1"), List.of("A2", "B2"), List.of("A2", "B3"));
+        List<List<String>> populatedStats = new ArrayList<>();
+        int numAValues = 10;
+        int numBValues = 2;
+        for (int indexA = 0; indexA < numAValues; indexA++) {
+            for (int indexB = 0; indexB < numBValues; indexB++) {
+                populatedStats.add(List.of("A" + indexA, "B" + indexB));
```

Comment:
Do you need the `populatedStats` list? Why not just do `cacheStatsHolder.incrementHits(List.of(...))` here?

Reply:
We don't need `populatedStats` after running `incrementHits`, but with the list + the for loop we don't have to explicitly write out `List.of(List.of("A0", "B0"), List.of("A0", "B1"), List.of("A1", "B0"), ... List.of("A9", "B1"))`.